# Force gnome-shell-41.9 to build with gsettings-desktop-schemas-40.0, even
# though it requires 41.0. We will provide the missing lockdown schema via
# a custom gschema.override file.
diff -Naur gnome-shell-41.9/meson.build gnome-shell-41.9-patched/meson.build
--- gnome-shell-41.9/meson.build	2022-11-07 22:12:51.545039805 -0600
+++ gnome-shell-41.9-patched/meson.build	2022-11-07 22:15:40.262049957 -0600
@@ -28,7 +28,7 @@
 gtk_req = '>= 3.15.0'
 mutter_req = '>= 41.0'
 polkit_req = '>= 0.100'
-schemas_req = '>= 41.alpha'
+schemas_req = '>= 40.alpha'
 startup_req = '>= 0.11'
 ibus_req = '>= 1.5.19'
 gnome_desktop_req = '>= 3.35.90'
